styles/main.css

```diff
@@ -60,6 +60,7 @@ footer {
 header nav ul {
     list-style: none;
     margin: 0;
+    padding: 0;

```

Comment:
Missing from previous lab; this causes your mobile menu items to be slightly off-center